.ci/integration-test.ts

```diff
@@ -3,5 +3,5 @@
 import { $ } from "zx";
 
 await $`docker compose -f ./docker-compose/docker-compose.test.yml up test-consumer-api test-admin-ui test-seed-mssql test-seed-client -d`;
-await $`dotnet test --no-restore --no-build "Backbone.sln" --filter "Category=Integration&TestCategory!~ignore"`;
-await $`docker compose -f ./docker-compose/docker-compose.test.yml down`;
\ No newline at end of file
+await $`dotnet test "Backbone.sln" --filter "Category=Integration&TestCategory!~ignore"`;
```

Comment:
Run separate `dotnet restore` and `dotnet build` commands. This will make the output easier to read.